m4_sessions/automation/session_one/one_elijah_annie_schutte_devin.md

```diff
@@ -1,3 +1,71 @@
 ## Group One's Session Plan
 
 M4 Student Leaders: Elijah, Annie, Schutte, Devin
+
+# Turing Career Development: Automation
+
+## Session Outcomes
+
+## Mission Alignment		 
+  		  
+*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
+  		  
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+
+## Pre-Session
+
+
+### Survey Questions
+
+Survey Questions to share with instructional staff by Wednesday of the same week.
+
+1.   What value would you assign to this session (from 1-5)?
+2.   What are you taking away from this session?
+3.   What role, if any, does Turing play in taking responsibility for this issue?
+
+
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
+#### Pros
+
+*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
+*   ["The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"](https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review) (_The Guardian_: October 1, 2015)
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+
+#### Cons
```

Comment:
love the pro/con list of resources. i'm concerned they won't be able to get to reading all of these though!